centaur/src/it/scala/centaur/AbstractCentaurTestCaseSpec.scala

```diff
@@ -26,11 +28,16 @@ abstract class AbstractCentaurTestCaseSpec(cromwellBackends: List[String], cromw
   SuccessReporters.getClass
 
   private def testCases(baseFile: File): List[CentaurTestCase] = {
+    val testsReports = TestsReportsSerializer.read(CentaurConfig.testsReportsPath)
+    val cromwellVer = VersionUtil.getVersion(CentaurConfig.cromwellName)
+    val skipPassedTests = CentaurConfig.skipPassedTests
+    val passedTestsNamesSet = if (skipPassedTests) testsReports.testsToSkip(cromwellVer) else Set.empty[String]
+
     val files = baseFile.list.filter(_.isRegularFile).toList
     val testCases = files.traverse(CentaurTestCase.fromFile(cromwellTracker))
 
     testCases match {
-      case Valid(l) => l
+      case Valid(l) => l.filterNot(x => passedTestsNamesSet.contains(getTestName(x)))
```

Comment:
It looks like this won't be needed if `skipPassedTests ` is false, so maybe we could check it here in order to remove unnecessary empty set and filter? What do you think?

Reply:
OK :+1: :